mirascope/core/vertex/call_response.py

```diff
@@ -0,0 +1,177 @@
+"""This module contains the `VertexCallResponse` class.
+
+usage docs: learn/calls.md#handling-responses
+"""
+
+from google.cloud.aiplatform_v1beta1.types import (
+    tool as gapic_tool_types,
```

Comment:
what is `gaipc` stand for? not sure how this is the shortened version of the import, should this be `gcaip`?

Reply:
<img width="660" alt="image" src="https://github.com/user-attachments/assets/282eb397-8fae-4ad9-a26e-19c5d82b5291">
To be honest, I don't know what it means, but I decided on this import policy in accordance with the sdk's internal naming conventions.

I will change it to `gcaip`

Reply:
oh interesting, yeah I think we should do something that makes sense or just avoid entirely unless necessary